libkbfs/tlf_handle_resolve.go

```diff
@@ -268,7 +274,21 @@ func (ruid resolvableID) resolve(ctx context.Context) (
 		return nameIDPair{}, keybase1.SocialAssertion{}, tlf.NullID, err
 	}
 	// TODO(KBFS-2621): get the TLF ID and display name for implicit
-	// teams. Blocked on CORE-6623.
+	// teams properly, using a proper service RPC rather than hacking
+	// the tlfIDGetter like this. Blocked on CORE-6623.
+	if doResolveImplicit(ctx) && ruid.id.IsTeamOrSubteam() &&
+		ruid.idGetter != nil {
+		// Passing in a nil handle will only work if we have a
+		// constIDGetter.  See the above TODO.
+		id, err := ruid.idGetter.GetIDForHandle(ctx, nil)
```

Comment:
This is a huge hack that I'll fix once CORE-6623 is done.  Seems to work for now though.